scripts/specs.ts

```diff
@@ -92,19 +93,25 @@ for (const [allowedUrl, message] of defaultAllowlist) {
 for (const [id, data] of Object.entries(features)) {
     const specs = Array.isArray(data.spec) ? data.spec : [data.spec];
     for (const spec of specs) {
-        const url = new URL(spec);
-        if (!isOK(url)) {
+        try {
+            var url = new URL(spec);
```

Comment:
```suggestion
            const url = new URL(spec);
```

Reply:
Ha, so I started to say something about this and forgot about it.

I think `var` is pretty confusing—it's weird that it's possible to reference it outside the scope here. Would you mind changing this a `let url` (where the original `const url = URL(spec);` was), then reassigning on this line?

```suggestion
            url = new URL(spec);
```

Reply:
Makes sense to me! Updated.